src/openrct2/actions/GameAction.h

```diff
@@ -208,15 +208,15 @@ struct GameAction
     using Callback_t = std::function<void(const struct GameAction*, const GameActionResult*)>;
 
 private:
-    uint32_t const _type;
+    GameCommand const _type;
```

Comment:
This might actually need to be a `uint32_t` if anything else uses `GameAction` - double check.

Reply:
This looks correct. _type will always be a GameCommand.